tests/Report/JsonReportTest.php

```diff
@@ -0,0 +1,150 @@
+
+        /* @var StopwatchEvent|\PHPUnit_Framework_MockObject_MockObject */
+        $mockEvent = $this->getMockBuilder('Symfony\Component\Stopwatch\StopwatchEvent')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $mockEvent
+            ->expects($this->once())
+            ->method('getMemory')
+            ->willReturn(2.5 * 1024 * 1024);
+        $mockEvent
+            ->expects($this->once())
+            ->method('getDuration')
+            ->willReturn(1234);
+
+        $this->mockStopwatch
+            ->expects($this->once())
+            ->method('getEvent')
+            ->with($this->equalTo('fixFiles'))
+            ->willReturn($mockEvent);
+
+        $this->mockOutput
+            ->expects($this->once())
+            ->method('getVerbosity')
+            ->willReturn(OutputInterface::VERBOSITY_QUIET);
+
+        $actualJson = '';
+        $expectedJson = '{"files":[{"name":"someFile.php"}],"memory":2.5,"time":{"total":1.234}}';
+
+        $this->mockOutput
+            ->expects($this->once())
+            ->method('write')
+            ->with($this->callback(function ($json) use (&$actualJson) { return $actualJson = $json; }));
+
+        $this->report->process(array(
+            'someFile.php' => array(
+                'appliedFixers' => array('some_fixer_name_here'),
+            ),
+        ));
+
+        $this->assertJsonStringEqualsJsonString($expectedJson, $actualJson);
+    }
+
+    public function testProcessWithDebugVerbosityAndDiffOption()
+    {
+        $this->mockInput
+            ->expects($this->once())
+            ->method('getOption')
+            ->with($this->equalTo('diff'))
+            ->willReturn(true);
+
+        /* @var StopwatchEvent|\PHPUnit_Framework_MockObject_MockObject */
+        $mockEvent = $this->getMockBuilder('Symfony\Component\Stopwatch\StopwatchEvent')
+            ->disableOriginalConstructor()
+            ->getMock();
+        $mockEvent
+            ->expects($this->once())
+            ->method('getMemory')
+            ->willReturn(2.5 * 1024 * 1024);
+        $mockEvent
+            ->expects($this->once())
+            ->method('getDuration')
+            ->willReturn(1234);
+
+        $this->mockStopwatch
+            ->expects($this->once())
+            ->method('getEvent')
+            ->with($this->equalTo('fixFiles'))
+            ->willReturn($mockEvent);
+
+        $this->mockOutput
+            ->expects($this->once())
+            ->method('getVerbosity')
+            ->willReturn(OutputInterface::VERBOSITY_DEBUG);
+
+        $actualJson = '';
+        $expectedJson = '{"files":[{"name":"someFile.php","appliedFixers":["some_fixer_name_here"],"diff":"this text is a diff ;)"}],"memory":2.5,"time":{"total":1.234}}';
+
+        $this->mockOutput
+            ->expects($this->once())
+            ->method('write')
+            ->with($this->callback(function ($json) use (&$actualJson) { return $actualJson = $json; }));
```

Comment:
No, One-liner are not allowed.
The appendix says that this is allowed:

```php
$app->get('/hello/{name}', function ($name) use ($app) { 
    return 'Hello '.$app->escape($name); 
});
```

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide-meta.md#31---multi-line-arguments-09082013